App/Sagas/NodeOnline.ts

```diff
@@ -26,5 +27,8 @@ export function * onNodeOnline () {
 
     // Check for new photos on every online event
     yield put(StorageActions.refreshLocalImagesRequest())
+
+    // Only run this after everything else in the node is running
+    yield call(runRecurringMigrationTasks)
```

Comment:
added here instead of lifecycle because it seems solid to just run it as the very last thing during a coming up event, since it is lowest priority

Reply:
Ok last thing here. The way this is set up, if you happen to get another `online` event while a previous `runRecurringMigrationTasks` saga is running, you're going to get repeated calls to the various apis for the same data. If that isn't harmful at all, cool. If you think it's a problem, this can be fixed by dispatching some `requestrunRecurringMigrationTasks` action and having a saga that listens for it using `take`, blocking while it yields the `runRecurringMigrationTasks` saga, then loops around to `take` again. You can see this in sagas i've written that start with `while (true) { yield take(...) ... }`